lib/ui/notifications.dart

```diff
@@ -24,39 +24,45 @@ class NotificationsWidgetState extends State<NotificationsWidget> {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      bottomNavigationBar: const BottomBar(),
-      appBar: TopBar(title: "Notifications"),
-      body: SingleChildScrollView(
-        child: SafeArea(
-          child: Column(
-            children: [
-              Container(
-                color: const Color.fromRGBO(228, 228, 228, 0.6),
-                child: Column(
-                  children: [
-                    Row(
-                      children: const [
-                        Expanded(
-                            child: Text(
-                              "Notifications",
-                              style: TextStyle(
-                                  fontSize: 20, fontWeight: FontWeight.bold),
-                            )),
-                        Expanded(
-                          child: Text(
-                            "Manage",
-                            style: TextStyle(
-                                fontSize: 20, fontWeight: FontWeight.bold),
-                          ))
-                      ],
-                    ),
-                  ],
-                ),
-              ),
-            ],
+    return DefaultTabController(
+      initialIndex: 1,
+      length: 2,
+      child: Scaffold(
+        bottomNavigationBar: const BottomBar(),
+        appBar: AppBar(
```

Comment:
Change this to just appBar: TopBar(title: "Notifications") to use shared topBar code

Reply:
Snippet came over weird. Just instead of initializing the AppBar, use TopBar and pass the desired title.

Reply:
little more complex to change here since he's using a TabBar as a part of this, probably will require changes to the constructor of TopBar to be able to supply this element

Reply:
Could probably pass it in as an optional parameter pretty easily

Reply:
Looked at it more and it does seem to have some weirder issues passing it that way, so I'll resolve this comment. Sorry to hold it up!